rbd.c

```diff
@@ -475,6 +475,19 @@ static void tcmu_rbd_close(struct tcmu_device *dev)
 	tcmu_rbd_state_free(state);
 }
 
+static int tcmu_rbd_handle_blacklisted_cmd(struct tcmu_device *dev,
+					   struct tcmulib_cmd *cmd)
+{
+       alua_lock_lost_notify(dev);
```

Comment:
Note. This is a temp hack.

We are going to redo the command completion code so instead of the handlers setting the sense we will just have them return a TCMUR_CMD_ERR_XYZ. runner will then update the lock state. This will be done in the next patchset to support the case where the iscsi/target path is ok, but the link to the cluster has gone bad. That will add code to handle command time outs and forced completion.